src/main/kotlin/nexters/admin/service/auth/AuthService.kt

```diff
@@ -0,0 +1,19 @@
+package nexters.admin.service.auth
+
+import nexters.admin.common.auth.JwtTokenProvider
+import nexters.admin.common.exception.UnauthenticatedException
+import nexters.admin.repository.MemberRepository
+import org.springframework.stereotype.Service
+
+@Service
+class AuthService(
+        val memberRepository: MemberRepository,
+        val jwtTokenProvider: JwtTokenProvider
```

Comment:
접근 제어자 명시해주면 좋을 듯합니다.
```suggestion
        private val memberRepository: MemberRepository,
        private val jwtTokenProvider: JwtTokenProvider
```

Reply:
반여완료